src/tree/SubscriptionTreeItem.ts

```diff
@@ -252,8 +252,9 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
                         }
                     }
 
+                    const tenantId = parent.subscription.tenantId ?? databaseAccount.identity?.tenantId;
```

Comment:
Workspace TreeVIew V2 doesn't have subscription. How to get tenantId having only connection string?